tests/test_filepath.py

```diff
@@ -169,6 +170,9 @@ def test_filepath_class(table=Filepath(), store="repo"):
     # delete from external table
     table.external[store].delete(delete_external_files=True)
     dj.errors._switch_filepath_types(False)
+    dj.config["filepath_checksum_size_limit"] = None
+    # Todo: assert here that the warning from external.download_file has been logged
+    # Waiting on DJ Logging implementation
```

Comment:
do not leave TODOs in the code. All issues should be tracked through the github issue manager.

Reply:
I have removed it and filed an issue. https://github.com/datajoint/datajoint-python/issues/1030